### PR TITLE
Better cyborg creation logging

### PR DESCRIPTION
**Pull request by dragomagol**
## About The Pull Request
A bunch of code improvements (better variable names, SECONDS defines, autodocs etc) plus more logging for newly-created cyborgs. Also cyborgs do not re-sync if they're already synced to the malf AI. 

## Why It's Good For The Game
Related to #44349 , we want to make sure we know who's synced to who. 

## Changelog

:cl: Tattle
admin: cyborg creation details (synced AI and default laws) are now logged
/:cl:

### CODE REVIEW

File: code/game/machinery/transformer.dm [hunk 1]
```diff
@@ -43,24 +50,22 @@
...
-		var/mob/living/carbon/human/H = AM
-		if((transform_standing || H.body_position == LYING_DOWN) && move_dir == EAST)// || move_dir == WEST)
-			AM.forceMove(drop_location())
-			do_transform(AM)
-
+		var/move_dir = get_dir(loc, entering_thing.loc)
+		var/mob/living/carbon/human/victim = entering_thing
+		if((transform_standing || victim.body_position == LYING_DOWN) && move_dir == EAST)
+			entering_thing.forceMove(drop_location())
+			do_transform(victim)
```
Comment (dragomagol) [inline code/game/machinery/transformer.dm:64]:
```suggestion
			do_transform(entering_thing)
```

File: code/game/machinery/transformer.dm [hunk 2]
```diff
@@ -71,43 +76,45 @@
...
 		return
-	if(cooldown == 1)
+
+	if(cooldown)
 		return
 
-	if(!transform_dead && H.stat == DEAD)
-		playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 50, FALSE)
+	if(!transform_dead && victim.stat == DEAD)
+		playsound(src.loc, 'sound/machines/buzz-sigh.ogg', 5 SECONDS, FALSE)
```
Comment (zxaber) [inline code/game/machinery/transformer.dm:87]: The third arg of playsound is volume, not sound play time.

File: code/game/machinery/transformer.dm [hunk 3]
```diff
@@ -71,43 +76,45 @@
...
+	victim.emote("scream") // It is painful
+	victim.adjustBruteLoss(max(0, 80 - victim.getBruteLoss())) // Hurt the human, don't try to kill them though.
 
 	// Sleep for a couple of ticks to allow the human to see the pain
 	sleep(0.5 SECONDS)
 
 	use_power(active_power_usage) // Use a lot of power.
-	var/mob/living/silicon/robot/R = H.Robotize()
-	R.cell = new /obj/item/stock_parts/cell/upgraded/plus(R, robot_cell_charge)
+	var/mob/living/silicon/robot/new_borg = victim.Robotize()
```
Comment (zxaber) [inline code/game/machinery/transformer.dm:103]: Also, borgs generally get their AI connection during `Initialize()`. Overwriting it after creating the borg should work fine, except that if another AI exists (on the same Z-level), it'll be getting alerts for new borgs before they get re-assigned. 

Possibly, `Robotize()` should be updated to handle receiving a ref to the Malf AI, and then passing it along to the new borg's `Initialize()`.

File: code/game/machinery/transformer.dm [hunk 4]
```diff
@@ -2,24 +2,31 @@
...
+	/// Whether we're on cooldown
+	var/cooldown = FALSE
+	/// How long until the next mob can be processed
 	var/cooldown_timer
+	/// The created cyborg's cell chage
 	var/robot_cell_charge = 5000
+	/// The visual countdown effect
 	var/obj/effect/countdown/transformer/countdown
+	/// Who the master AI is that created this factory
 	var/mob/living/silicon/ai/masterAI
```
Comment (ZephyrTFA) [inline code/game/machinery/transformer.dm:27]: want to change this var name while you're here? 🥺

Comment (dragomagol) [reply code/game/machinery/transformer.dm:27]: What do you suggest?

Comment (ZephyrTFA) [reply code/game/machinery/transformer.dm:27]: master_ai

File: code/game/machinery/transformer.dm (hunk 3 above)
Comment (dragomagol) [reply code/game/machinery/transformer.dm:103]: I was definitely thinking about doing this when I made the PR, I'll look into it!

Comment (dragomagol) [reply code/game/machinery/transformer.dm:103]: On second thought, leaving this for another PR.

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 5]
```diff
@@ -61,7 +61,7 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
 /datum/action/innate/ai/Grant(mob/living/L)
```
Comment (jlsnow301) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm:61]: could you change L to `player`?

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 6]
```diff
@@ -774,27 +780,27 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
...
 /datum/ai_module/upgrade/upgrade_cameras/upgrade(mob/living/silicon/ai/AI)
 	AI.lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE //Night-vision, without which X-ray would be very limited in power.
 	AI.update_sight()
 
 	var/upgraded_cameras = 0
-	for(var/obj/machinery/camera/C as anything in GLOB.cameranet.cameras)
-		var/obj/structure/camera_assembly/assembly = C.assembly_ref?.resolve()
+	for(var/obj/machinery/camera/camera as anything in GLOB.cameranet.cameras)
+		var/obj/structure/camera_assembly/assembly = camera.assembly_ref?.resolve()
 		if(assembly)
```
Comment (jlsnow301) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm:793]:
```
		if(!assembly)
		    continue
```
(rest would need unindented)

**Review by jlsnow301 — CHANGES_REQUESTED** [review]

**Pull request merged**